CHANGELOG.latest.md

```diff
@@ -1,6 +1,28 @@
-### RevenueCatUI
-* `Paywalls`: add `PaywallFooterViewController` (#3486) via Toni Rico (@tonidero)
-* `Paywalls`: improve landscape support of all templates (#3471) via NachoSoto (@NachoSoto)
-* `Paywalls`: ensure footer links open in full-screen sheets (#3524) via NachoSoto (@NachoSoto)
-* `Paywalls`: improve `FooterView` text alignment (#3525) via NachoSoto (@NachoSoto)
-* Paywalls: Add dismissal method in `PaywallViewControllerDelegate` (#3493) via Toni Rico (@tonidero)
+### StoreKit 2 Beta
+
+This beta introduces a new configuration option to enable full StoreKit 2 flow on the SDK and the RevenueCat backend.

```

Comment:
let's add some extra clarification between what the new version of SK2 support is vs the old one. We internally call it "full" StoreKit 2, but the distinction probably wont' be immediately clear to a customer, who could reasonably wonder what this means if they've had a storekit 2 flag for years

Reply:
That'll also help explain and justify the deprecation of the old flag